crates/miden-lib/src/account/components/mod.rs

```diff
@@ -244,7 +241,7 @@ impl WellKnownComponent {
     /// Gets all well known components which could be constructed from the provided procedures map
     /// and pushes them to the `component_interface_vec`.
     pub fn extract_well_known_components(
-        procedures_map: &mut BTreeMap<Word, &AccountProcedureInfo>,
+        procedures_map: &mut BTreeMap<Word, &AccountProcedureRoot>,
```

Comment:
Similar comment as above - I think this can be just `BTreeMap<AccountProcedureRoot>`.